systers_portal/templates/base.html

```diff
@@ -37,17 +37,12 @@
     <div class="collapse navbar-collapse">
       <ul class="nav navbar-nav text-uppercase" id='lower-nav'>
         {% url 'about-us' as url %}
-        <li {% if url == request.path %}class="active"{% endif %}>
+        <li id="nav-menu" {% if url == request.path %}class="active"{% endif %}>
           <a href="{{ url }}">About Us</a>
         </li>
-        <li class="dropdown {% if '/community/' in request.path or url == request.path %}active{% endif %}" id="community-dropdown">
+        <li id="nav-menu" class="dropdown {% if '/community/' in request.path or url == request.path %}active{% endif %}" id="community-dropdown">
```

Comment:
This is not really needed, and I find the way it is done through Django templates not elegant.